01-introduction.Rmd

```diff
@@ -113,121 +167,140 @@ Because R is open source, these developments can easily build on previous work,
 This approach is advantageous because it encourages collaboration and avoids 'reinventing the wheel'.
 The package **sf** (covered in Chapter \@ref(spatial-class)), for example, builds on its predecessor **sp**.
 
-Shifts in R's spatial ecosystem have been influenced by shifts in the wider R community, as exemplified by the visualization and data processing packages **ggplot2** and **dplyr** (released in [2007](https://cran.r-project.org/src/contrib/Archive/ggplot2/) and [2014](https://cran.r-project.org/src/contrib/Archive/dplyr/) respectively).
```

Comment:
I think this subsection is better placed a bit below (see below :)). 
Additionally, I would not say that ggplot2 has initiated any shift in the R-spatial world. But maybe that's just a personal opinion.
Please note also that Edzer is rather sceptical with regard to all the tidyverse-stuff.